src/aui-form-builder/assets/aui-form-builder-field-list-core.css

```diff
@@ -2,50 +2,86 @@
     background-color: #ecf6fe;
     border: 1px solid #65b3f0;
     border-radius: 2px;
-    display: none;
-    margin: 0 auto;
-    opacity: 0.5;
+    visibility: hidden;
     text-align: center;
-    width: 92%;
+    cursor: pointer;
 }
 
 .form-builder-field-list-add-button-icon {
-    background-color: #108BD1;
+    background-color: #65b3f0;
     border-radius: 50%;
     font-size: 24px;
     font-weight: 200;
-    height: 30px;
+    height: 32px;
     margin: 0 auto;
-    width: 30px;
+    width: 32px;
+    cursor: pointer;
 }
 
 .form-builder-field-list-add-button-label {
-    color: #4a90e2;
+    color: #66a1cc;
     font-family: "Helvetica Neue";
     font-size: 11px;
+    margin-bottom: 0;
+    cursor: pointer;
+    opacity: .5;
 }
 
 .form-builder-field-list-add-button-plus-icon {
     color: white;
-    line-height: 27px;
-    margin-top: 8px;
+    line-height: 28px;
+    margin-top: 20px;
+    margin-bottom: 31px;
 }
 
 .form-builder-field-list-add-button-visible .form-builder-field-list-add-button {
-    display: block;
+    visibility: visible;
+}
+
+.form-builder-field-list-add-button:hover .form-builder-field-list-add-button-label {
+    opacity: 1;
+}
+
+.form-builder-field-list-empty .form-builder-field-list-add-button {
+    opacity: 1;
+}
+
+.form-builder-field-list-empty .form-builder-field-list-add-button-icon {
+    opacity: .5;
+}
+
+.form-builder-field-list-empty .form-builder-field-list-add-button-icon:hover {
+    opacity: 1;
+}
+
+.form-builder-field-list-empty .form-builder-field-list-add-button-label {
+    margin-bottom: 18px;
+    margin-top: 9px;
+}
+
+.form-builder-field-list-empty .form-builder-field-list-add-container {
+    position: initial;
+}
+
+.form-builder-field + .form-builder-field-list-add-container {
```

Comment:
@thiago-rocha, how does Alloy dial with progressive enhancement? This selector above, for example, it isn't supported by IE < 8. It doesn't mean that the form builder won't works on these browser version, just the alignment of a element won't be ok.